elasticsearch/full-text/src/main/java/nl/knaw/huc/service/FieldsService.java

```diff
@@ -51,13 +51,19 @@ public Fields createFields(
   private Fields fromTxt(@NotNull InputStream inputStream) {
     String text;
     try {
-      text = IOUtils.toString(inputStream, UTF_8);
+      text = this.cleanUp(IOUtils.toString(inputStream, UTF_8));
```

Comment:
Is `this.` hier echt nodig voor lokale method-call? Is er een andere `cleanUp` die voor verwarring zorgt?
Misschien 'overzealous' resultaat van een IntelliJ instelling die bij 'refactor > extract method' afgaat?

Reply:
d700baa